code/modules/food/recipes_microwave.dm

```diff
@@ -1170,8 +1170,41 @@ I said no!
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/spicy_clams
 
-
 /datum/recipe/earthenroot_soup
 	fruit = list("earthenroot" = 2)
 	reagents = list("water" = 10, "spacespice" = 1, "sodiumchloride" = 1)
-	result = /obj/item/weapon/reagent_containers/food/snacks/earthenroot_soup
\ No newline at end of file
+	result = /obj/item/weapon/reagent_containers/food/snacks/earthenroot_soup
+
+/datum/recipe/onionrings
+	fruit = list("onion" = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/doughslice
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/onionrings
+
+/datum/recipe/berrymuffin
+	reagents = list("milk" = 5, "sugar" = 5)
+	reagent_mix = RECIPE_REAGENT_REPLACE
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/dough
+	)
+	fruit = list("berries" = 1)
+	result = /obj/item/weapon/reagent_containers/food/snacks/muffin
+
+/datum/recipe/onionsoup
+	fruit = list("onion" = 1)
+	reagents = list("water" = 10)
+	result = /obj/item/weapon/reagent_containers/food/snacks/onionsoup
```

Comment:
The path of the soup needs to be changed to
`/obj/item/weapon/reagent_containers/food/snacks/soup/beet`
to be compliant with the path changes in the cargo reward update.

Reply:
And I screwed up the copy/pasted path.
It should be /soup/onion and not soup/beet